test/name.spec.js

```diff
@@ -99,7 +99,7 @@ describe('.name', () => {
         .then((res) => {
           expect(res).to.exist
           expect(res).to.be.eql({
-            Path: '/ipfs/' + name.Value
+            Path: name.Value
```

Comment:
@diasdavid I think it is now actually returning the value with `/ipfs/` in the path. The two lines here starting with `>` are the result value from go-ipfs http request:

```
    Promise API
> { Name: 'QmQ8qViFq8ke3vQVV3Qfb84GwPxmGAUm1a8TzDPD3u6GM1',
  Value: '/ipfs/Qma4hjFTnCasJ8PVp3mZbZK5g2vGDT4LByLJ7m8ciyRFZP' }
      ✓ .name.publish
> { Name: 'QmQ8qViFq8ke3vQVV3Qfb84GwPxmGAUm1a8TzDPD3u6GM1',
  Value: '/ipfs/Qma4hjFTnCasJ8PVp3mZbZK5g2vGDT4LByLJ7m8ciyRFZP' }
      1) .name.resolve
```

Is it the case that previously it returned just the hash without the `/ipfs/` in front and now it's returning it correctly (as expected in the tests, ie. we don't need to prepend it to the return value anymore).